### PR TITLE
Package portia.1.5

### PR DESCRIPTION
**Pull request by rixed**

### CODE REVIEW

File: packages/portia/portia.1.5/opam [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  "ocamlfind" {build}
+  "batteries"
+  "qtest" {with-test}
+]
+depexts: [
+  ["asciidoc"] {os-family = "debian"}
+]
+post-messages: [
+  "This package requires asciidoc to build the doc." {failure}
+]
```
Comment (kit-ty-kate) [inline packages/portia/portia.1.5/opam:36]: Do you have time to create a new `conf-asciidoc` package? This way the documentation would be available everywhere, maintainance would be reduced and the dependency would be removed during normal execution (non-doc).

If you don't have time I'll do it at some point

Comment (rixed) [reply packages/portia/portia.1.5/opam:36]: I can not look into this in the near future but maybe in a few weeks.

Comment (kit-ty-kate) [reply packages/portia/portia.1.5/opam]: That's ok, I've just done it ^^

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**